services/kanto/src/kanto_service.cpp

```diff
@@ -3,6 +3,8 @@
 #include "system_ability_definition.h"
 #include <unistd.h>
 #include <system_error>
+#include "mqtt/client.h"
+#include "json.hpp"
```

Comment:
Where does the json.hpp file come from? Gitlab web view does refuse to load it as it is to big with 24K lines. Origin and license would be interesting for the this. I assume its a one file json parser.